Anytype/Sources/ServiceLayer/Configuration/AnytypeErrorLoggerConfiguration.swift

```diff
@@ -1,15 +1,19 @@
 import Foundation
 import Sentry
 
+@MainActor
 protocol AppErrorLoggerConfigurationProtocol: AnyObject {
-    func setUserId(_ userId: String)
+    func setUserId(_ userId: String) async
```

Comment:
Why is this method marked as async and configuration as MainActor ?

Reply:
we got warning after sentry update
<img width="1059" alt="Screenshot 2024-04-10 at 16 39 30" src="https://github.com/anyproto/anytype-swift/assets/6065952/03113644-c068-4c44-95cb-fb4ce1ad9d08">

Reply:
remove async